charts/tekton-apps/templates/general/argocd_applications.yaml

```diff
@@ -23,7 +23,7 @@ metadata:
     jiraURL: {{ $project.jiraURL }}
 spec:
   destination:
-    namespace: {{ $project.argocd.namespace }}
+    namespace: {{ or ($component.argocd).namespace ($project.argocd).namespace | required ".component.namespace is required in project settings" }}
```

Comment:
Pls add example to docs

Reply:
Added example https://github.com/saritasa-nest/saritasa-devops-helm-charts/commit/7db6f2c8de806bac83063cf56908256382a8b091